### ISSUE TITLE
User interface to enter alt text for images

### ISSUE THREAD

**Issue by michael** [michael/editable-website#93]
Challenge here is that alt-text is not entered on the canvas. So we'll need some sort of overlay (similar to link editing). Since MediaControls is already a bit busy the overlay must not occupy any space inside the media itself.

I'd try first to have a similar flow as with links. When there's no alt-text "Add alt text", when there is alttext, preview it, with an EDIT button next to it.

There might be potential conflict with link editing when there's an image inside al inkable surface. Consider this selection:

<img width="480" height="491" alt="Image" src="https://github.com/user-attachments/assets/db97183d-0cab-4935-bb8e-bc988ac47355" />

Not sure how to solve this, because we might need to show both overlays at the same time. Because you might just have an image that is linked. So you need link controls and image controls.

**Comment by michael** [comment]
Done.
